### ISSUE TITLE
(Longer term) Expand discussion of PBC to address visualization issues with "molecules outside the box"

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/basic_simulation_training#93]
If I had $1 for every time I've had to explain why molecules appear "outside the box" on visualization, I'd be quite wealthy. In a subsequent revision we should add a discussion of this to the section on periodic boundary conditions, as it's an issue that foils a lot of people.

(I've actually seen the problem cut both ways, too -- sometimes people assume it is NOT a problem when it is, because they've messed up something in their analysis. But more often it's that people think it is a huge problem when it's just an issue of visualization.)

**Comment by justinGilmer** [comment]
Ive found when explaining it to students, that video games are a pretty great way to visualize it.

Pacman is a nice example of pbcs, leave one side of the map, enter in the exact same orientation like you never left the box to begin with, on the other side of the map

Although this does not help explain when the molecules appear to diffuse outside of the box

Its a bit harder to explain the who its interaction neighbors are (MIC), but at least for the visualization purposes, it helps.

**Comment by davidlmobley** [comment]
Yes, I do use this analogy in class, but need to get it into te written materials.
